starport/cmd/tools.go

```diff
@@ -85,3 +86,67 @@ func toolsProxy(ctx context.Context, command []string) error {
 		),
 	)
 }
+
+func NewToolsCompletions() *cobra.Command {
+
+	// completionCmd represents the completion command
+	c := &cobra.Command{
+		Use:   "completions",
+		Short: "Generate completions script",
+		Long: `To load completions:
+
+Bash:
+
+  $ source <(starport  tools completions bash)
+
+  # To load completions for each session, execute once:
+  # Linux:
+  $ starport  tools completions bash > /etc/bash_completion.d/starport
+  # macOS:
+  $ starport  tools completions bash > /usr/local/etc/bash_completion.d/starport
+
+Zsh:
+
+  # If shell completions is not already enabled in your environment,
+  # you will need to enable it.  You can execute the following once:
+
+  $ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+  # To load completions for each session, execute once:
+  $ starport  tools completions zsh > "${fpath[1]}/_starport"
+
+  # You will need to start a new shell for this setup to take effect.
```

Comment:
we can apply the changes in the current shell running `source ~/.zshrc`

Reply:
@Pantani, this can raise a lot of possibilities, so the best strategy is to follow in the footsteps of [kubectl](https://kubernetes.io/docs/tasks/tools/included/optional-kubectl-configs-bash-linux/) and address this issue in the documentation. I'll open an new issue for that.